src-cpp/rdkafkacpp_int.h

```diff
@@ -498,11 +554,22 @@ class ConfImpl : public Conf {
       return Conf::CONF_OK;
     }
 
+  Conf::ConfResult get(CertVerifyCb *&ssl_cert_verify_cb) const {
```

Comment:
there is a mix of cert_verify_ and ssl_cert_verify_.., I would prefer the latter both for the callbacks as for the classes to make it absolutely clear that this is Ssl Certificates and nothing else.